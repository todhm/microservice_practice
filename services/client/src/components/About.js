import React from "react";

export default () => (
  <div>
    <h1 className="title is-1">About</h1>
    <hr />
    <br />
    <p>Add something relevant here.</p>
  </div>
);
