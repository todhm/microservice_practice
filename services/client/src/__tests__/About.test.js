import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";

import About from "../components/About";

test("About renders properly", () => {
  const wrapper = shallow(<About />);
  const element = wrapper.find("p");
  expect(element.length).toBe(1);
  expect(element.text()).toBe("Add something relevant here.");
});

test("About renders a snapshot properly", () => {
  const tree = renderer.create(<About />).toJSON();
  expect(tree).toMatchSnapshot();
});
