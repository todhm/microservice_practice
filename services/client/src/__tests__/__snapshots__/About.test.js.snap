// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders a snapshot properly 1`] = `
<div>
  <h1
    className="title is-1"
  >
    About
  </h1>
  <hr />
  <br />
  <p>
    Add something relevant here.
  </p>
</div>
`;
